Dockerfile

```diff
@@ -1,18 +1,14 @@
+ARG TAG="master"
 FROM python:3.7
 
-RUN python3 -m pip install requests flask gunicorn coverage 
+ENV FLASK_APP=art_to_ftp.py
+ENV APP_PORT=${APP_PORT:-5700}
 
-COPY . /local/art_to_ftp
-WORKDIR /local/art_to_ftp
+RUN python3 -m pip install flask gunicorn oc-cdtapi
 
-#RUN python3 -m coverage run -m unittest discover test && \
-#    mkdir -p /build/reports && \
-#    python3 -m coverage xml --include=./app/* -o /build/reports/art_to_ftp_coverage.xml
+WORKDIR /app
+COPY . /app
 
-RUN python3 -m unittest discover
-
-HEALTHCHECK --interval=1m --timeout=30s --start-period=15s --retries=3 \
-    CMD curl -v --silent http://localhost:5700/ping 2>&1 | grep '< HTTP/1.1 200'
-
-CMD /usr/local/bin/gunicorn wsgi:app --log-level=debug -b 0.0.0.0:5700
+#HEALTHCHECK --interval=300s --timeout=3s CMD ./healthcheck.sh ${APP_PORT} || exit 1
 
+CMD gunicorn -b 0.0.0.0:${APP_PORT} wsgi:app
```

Comment:
Pease use *ENTRYPOINT* instead of *CMD*, as we do in another projects:
https://github.com/octopusden/octopus-oc-srv-jobs-client-provider/blob/main/Dockerfile